src/ai/backend/manager/api/auth.py

```diff
@@ -951,12 +985,75 @@ async def update_password(request: web.Request, params: Any) -> web.Response:
         data = {
             "password": params["new_password"],
             "need_password_change": False,
+            "password_changed_at": datetime.utcnow(),
         }
         query = users.update().values(data).where(users.c.email == email)
         await conn.execute(query)
     return web.json_response({}, status=200)
 
 
+@check_api_params(
+    t.Dict(
+        {
+            t.Key("domain"): t.String,
+            t.Key("email"): t.String,
+            t.Key("old_password"): t.String,
+            t.Key("new_password"): t.String,
+            t.Key("new_password2"): t.String,
```

Comment:
Is it necessary to include `new_password2` in the API? I think that `new_password` is enough. Confirming the password is not a concern of the API.

Reply:
I agree with you. I tried to match the parameters with existing `update_password` API but it is cumbersome. I updated it!